simulator/src/simulator/simulation-node.ts

```diff
@@ -33,7 +33,7 @@ import {
 import { sleep } from './misc'
 import { getLatestBlockHash } from './utils/chain'
 
```

Comment:
We should parameterize this or make it somehow global definition of this root repo (so build is used and not installed binary)